src/ui/components/AccountSelector.js

```diff
@@ -1,47 +1,39 @@
 // @flow
-
 import React from "react";
 import {createMuiTheme, ThemeProvider} from "@material-ui/core/styles";
 import {TextField} from "@material-ui/core";
-import {format} from "../../ledger/grain";
 import {Autocomplete} from "@material-ui/lab";
 import {Ledger, type Account} from "../../ledger/ledger";
 
 type DropdownProps = {|
   +ledger: Ledger,
-  +setCurrentIdentity: (Account | null) => void,
+  +setCurrentAccount: (Account | null) => void,
   +placeholder?: string,
 |};
 
 const theme = createMuiTheme({
   overrides: {
-    // Name of the rule
     text: {
-      // Some CSS
       color: "black",
     },
   },
 });
 
 export default function AccountDropdown({
   placeholder,
-  setCurrentIdentity,
+  setCurrentAccount,
   ledger,
 }: DropdownProps) {
   const items = ledger.accounts().filter((a) => a.active);
 
-  const onComboChange = (e, inputObj) => setCurrentIdentity(inputObj);
-
   return (
     <ThemeProvider theme={theme}>
       <Autocomplete
-        onChange={onComboChange}
+        onChange={(e, inputObj) => setCurrentAccount(inputObj)}
```

Comment:
I'm surprised that the unused variable `e` isn't producing an eslint error.

Reply:
even when using the onComboChange as before there is an unused variable. Not sure how to get around this, as those are the default arguments passed, and I only need to use the 2nd argument.

Reply:
It looks like there's a way to do this with array destructuring... (...[ , inputObj]), but it's kind of ugly and I've never really seen that used in this context

Reply:
Okay, this is what I'm settling on I think:
onChange={(...args) => {
        const inputObj = args[1];
        setCurrentAccount(inputObj)
      }}

Reply:
this is the most elegant way to do the thing, and it works, but Flow throws an error about the rest parameter being unexpected:
onChange={(...[ , inputObj]) => setCurrentAccount(inputObj)}

Reply:
Use:

```js
onChange={(_, inputObj) => {
     setCurrentAccount(inputObj);
}}
```

Our eslint rules explicitly allow _ as an unused variable:
https://github.com/sourcecred/sourcecred/blob/9eec13c799b8b00e24c83279955191b5a6cc7623/.eslintrc.js#L30-L36